webvtt.go

```diff
@@ -38,6 +39,35 @@ func parseDurationWebVTT(i string) (time.Duration, error) {
 	return parseDuration(i, ".", 3)
 }
 
+// https://tools.ietf.org/html/rfc8216#section-3.5
+func computeOffsetFromTimestampMap(line string) (time.Duration, error) {
+	var timeOffset time.Duration
+
+	right := strings.Split(line, "=")[1]
+	var local time.Duration
+	var mpegts int64
+	for _, split := range strings.Split(right, ",") {
+		splits := strings.SplitN(split, ":", 2)
+		switch splits[0] {
```

Comment:
could you switch on `string.ToLowerCase(splits[0])` instead ? and replace `case` values with their lower case version ? that way the lib will handle `LOCAL` as well as `local` for instance